OBAKit/Stops/StopViewController.swift

```diff
@@ -278,19 +259,155 @@ public class StopViewController: UIViewController,
+            alertsAction.attributes = .disabled
+        }
+
+        return UIMenu(title: "File", options: .displayInline, children: [bookmarkAction, alertsAction])
+    }
+
+    fileprivate func locationMenu() -> UIMenu {
+        let nearbyAction = UIAction(title: OBALoc("stops_controller.nearby_stops", value: "Nearby Stops", comment: "Title of the row that will show stops that are near this one."), image: UIImage(systemName: "location")) { _ in
+            let nearbyController = NearbyStopsViewController(coordinate: self.stop!.coordinate, application: self.application)
+            self.application.viewRouter.navigate(to: nearbyController, from: self)
+        }
+
+        var walkingDirectionActions: [UIMenuElement] = []
+
+        if let stop = self.stop {
+            if let appleMapsURL = AppInterop.appleMapsWalkingDirectionsURL(coordinate: stop.coordinate) {
+                let appleMaps = UIAction(title: OBALoc("stops_controller.walking_directions_apple", value: "Walking Directions (Apple Maps)", comment: "Button that launches Apple's maps.app with walking directions to this stop")) { _ in
+                    self.application.open(appleMapsURL, options: [:], completionHandler: nil)
+                }
+                walkingDirectionActions.append(appleMaps)
+            }
+
+            #if !targetEnvironment(simulator)
+            // Display Google Maps app link, only if Google Maps is installed.
+            if let googleMapsURL = AppInterop.googleMapsWalkingDirectionsURL(coordinate: stop.coordinate),
+               self.application.canOpenURL(googleMapsURL) {
+                let googleMaps = UIAction(title: OBALoc("stops_controller.walking_directions_google", value: "Walking Directions (Google Maps)", comment: "Button that launches Google Maps with walking directions to this stop")) { _ in
+                    self.application.open(googleMapsURL, options: [:], completionHandler: nil)
+                }
+                walkingDirectionActions.append(googleMaps)
+            }
+            #endif
+        }
+
+        let walkingDirectionsElement: UIMenuElement
+        let walkingDirectionsTitle = OBALoc("stops_controller.walking_directions", value: "Walkign Directions", comment: "Button that launches a maps app with walking directions to this stop")
+        let walkingDirectionsImage = UIImage(systemName: "figure.walk")
+
+        // Show a disabled walking directions button if there are no Walking Directions apps available.
+        if walkingDirectionActions.isEmpty {
+            walkingDirectionsElement = UIAction(title: walkingDirectionsTitle, image: walkingDirectionsImage, attributes: .disabled) { _ in /* noop */ }
+        } else {
+            walkingDirectionsElement = UIMenu(title: walkingDirectionsTitle, image: walkingDirectionsImage, children: walkingDirectionActions)
+        }
+
+        return UIMenu(title: "Location", options: .displayInline, children: [nearbyAction, walkingDirectionsElement])
```

Comment:
Is this title shown anywhere? Should this be localized?

Reply:
doesn't need to be localized. I don't think this is user visible, except maybe in catalyst. The titles are mostly comments for me, in case of future expandability.

Reply:
Right on.